plugins/infinispan90/pom.xml

```diff
@@ -124,6 +124,17 @@
          <optional>true</optional>
       </dependency>
 
+      <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-marshaller-kryo</artifactId>
+         <version>${version.infinispan}</version>
```

Comment:
@mgencur until 9.0.0.Final is out, you should use `-Dinfinispan90.infinispan.version=9.0.0.Beta1` for build.